addons/crm/views/crm_lead_views.xml

```diff
@@ -126,6 +126,7 @@
                                 </div>
                                 <field name="website" widget="url" placeholder="e.g. https://www.odoo.com"/>
                                 <field name="lang_id"/>
+                                <field name="is_partner_visible" invisible='1'/>
```

Comment:
Could we add this new field _before_ "partner_id", since it's clearly related to it ?